.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "python.linting.pylintEnabled": true,
     "python.formatting.provider": "yapf",
-    "restructuredtext.confPath": ""
+    "restructuredtext.confPath": "",
+    "python.pythonPath": "/Users/leopold/Applications/miniconda3/envs/aiida-lsmo-2/bin/python"
```

Comment:
sorry about this, will delete.
can I prevent my vscode from doing this?

Reply:
maybe the folder should remain committed but be in the .gitignore?

Reply:
I didn't add it to gitignore based on https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control but since we already have the pre-commit hooks we actually do not really need it

Reply:
I see, thanks.
It seems there is the option to add `.default` files for those config files that are likely to change between installations